src/base/projects/Issue/IssueComment.svelte

```diff
@@ -18,11 +18,22 @@
   let profile: Profile | null = null;
 
   onMount(async () => {
-    if (comment.author.identity.ens?.name) {
+    if (comment.author.kind === "resolved" && comment.author.identity.ens?.name) {
       profile = await Profile.get(comment.author.identity.ens.name, ProfileType.Minimal, config);
     }
   });
 
+  $: source =
+    profile?.avatar ||
+    (comment.author.kind === "resolved"
+      ? comment.author.identity.urn
+      : comment.author.urn);
```

Comment:
Ah I see why. I would move the `URN` to the same place for both identity types to avoid this kind of issue.

Reply:
Strictly we don't need it to be resolved but since it could be either "resolved" or "unresolved", we need to check both variants to get always a `string` for `source`

Reply:
> Ah I see why. I would move the `URN` to the same place for both identity types to avoid this kind of issue.

👌  this would also work

Reply:
@cloudhead Should we move the `URN` maybe on the http-api? Seems like it's easier to handle it there..

Maybe we could do something like this
```rs
pub enum Identity {
    /// Only the URN is known.
    Unresolved { urn: Urn },
    /// The full user identity is resolved.
    Resolved { urn: Urn, identity: project::PeerIdentity },
}
```